src/App/Pages/LoginTwoFactorPage.cs

```diff
@@ -453,7 +460,7 @@ protected override void OnDisappearing()
                             break;
                         case TwoFactorProviderType.YubiKey:
                             var nfcKey = p.Value.ContainsKey("Nfc") && (bool)p.Value["Nfc"];
-                            if(!_deviceInfoService.NfcEnabled || !nfcKey)
+                            if((!_deviceInfoService.NfcEnabled || !nfcKey) && (Device.RuntimePlatform != Device.UWP))
```

Comment:
You dont need parens around your UWP check. Also dont you need some kind of "Has USB Port" check?